internal/server/providers/contracts/contracts_e2e_test.go

```
@@ -0,0 +1,499 @@
+	AskedAt     string  `json:"askedAt"`
+	Deadline    *string `json:"deadline,omitempty"`
+	BlocksClose bool    `json:"blocksClose"`
+}
+
+func postQuery(t *testing.T, url, query string) graphqlResponse {
+	t.Helper()
+	body, err := json.Marshal(map[string]string{"query": query})
+	if err != nil {
+		t.Fatalf("marshal: %v", err)
+	}
+	req, err := http.NewRequest(http.MethodPost, url+"/graphql", strings.NewReader(string(body)))
```

golangci-lint (2.12.1)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)